desktop/src/main/java/bisq/desktop/main/support/dispute/agent/SignedOfferView.java

```diff
@@ -318,7 +321,7 @@ public TableCell<SignedOffer, SignedOffer> call(TableColumn<SignedOffer, SignedO
                             @Override
                             public void updateItem(final SignedOffer item, boolean empty) {
                                 super.updateItem(item, empty);
-                                
+                                //TODO(niyid) Should be download button?
```

Comment:
Do you mean the ability to copy to clipboard? This would be a great feature to add.